post_processing/single_file_post_processing.py

```diff
@@ -348,33 +124,33 @@ def calculate_probabilities_and_thresholds(
     # find the rows where the proportion is >= pct_runs_under_threshold, select the top row as the
     # first index
     # todo: verify/dynamically select the lowest year
-    year_of_90_under_threshold = find_year_reaching_threshold(
+    year_of_pct_runs_under_threshold = _find_year_reaching_threshold(
```

Comment:
🚫  now you've pulled this out into its own module, it should _not_ be private (i.e. starting with an `_` - I'm surprised ruff doesn't generate an error for this.  By pulling it into a separate module I think it is no longer confusing either, this is a reasonable function to use if you want to work out a specific measure. Not sure if I'm making sense - happy to have a chat!